src/mangata-rococo.js

```diff
@@ -0,0 +1,161 @@
+import '@oak-network/api-augment';
+import _ from 'lodash';
+import moment from 'moment';
+import { cryptoWaitReady } from '@polkadot/util-crypto';
+import Keyring from '@polkadot/keyring';
+import confirm from '@inquirer/confirm';
+import TuringHelper from './common/turingHelper';
+import MangataHelper from './common/mangataHelper';
+import Account from './common/account';
+import { delay, listenEvents, readMnemonicFromFile } from './common/utils';
+import {
+    TuringStaging, MangataRococo,
+} from './config';
+
+/**
+ * README!
+ *
+ * 1. Unlike dev environment there’s no Sudo in Rococo, so we need to important a wallet first.
+ * a) Export a wallet from polkadot.js plugin, and save it as a file ./private/seed.json
+ * b) Run "PASS_PHRASE=<password_for_unlock> npm run mangata-rococo"
+ *
+ * 2. You need to have balances in both MGR and TUR on Mangata Rococo
+ * 3. You need to have balances in TUR on Turing Staging
+ */
+
+// Create a keyring instance
+const keyring = new Keyring({ type: 'sr25519' });
+
+/** * Main entrance of the program */
+async function main() {
+    await cryptoWaitReady();
+
+    console.log('Initializing APIs of both chains ...');
+    const turingHelper = new TuringHelper(TuringStaging);
+    await turingHelper.initialize();
+
+    const mangataHelper = new MangataHelper(MangataRococo);
+    await mangataHelper.initialize();
```

Comment:
The initialziation of helpers changed. In the beginning we pass configs into providers, so that they are tied to and can be read from providers.